submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -17,10 +22,39 @@ class SubmissionSubmitter(
         submission: ExtendedSubmission,
         files: FilesSource,
         context: PersistenceContext
+
     ): Submission {
-        validators.forEach { validator -> validator.validate(submission, context) }
-        processors.forEach { processor -> processor.process(submission, context) }
-        filesHandler.processFiles(submission, files)
+        val validationNodes = mutableListOf<ValidationNode>()
```

Comment:
I would suggest using a context object to store the errors if they exist, something like this approach:

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/tsv/deserialization/chunks/ChunkProcessor.kt#L51

And pass that object to each validator/processor.

Reply:
"Try"ing to minimize code change :D